stdlib/public/SDK/SceneKit/SceneKit.swift

```diff
@@ -29,9 +29,9 @@ extension SCNVector3 {
     self.z = SCNFloat(z)
   }
   public init(_ x: CGFloat, _ y: CGFloat, _ z: CGFloat) {
-    self.x = SCNFloat(x)
-    self.y = SCNFloat(y)
-    self.z = SCNFloat(z)
+    self.x = SCNFloat(x as NSNumber)
+    self.y = SCNFloat(y as NSNumber)
+    self.z = SCNFloat(z as NSNumber)
```

Comment:
Oh, I don't think that was intended.

```
#if os(OSX)
public typealias SCNFloat = CGFloat
#elseif os(iOS) || os(tvOS)
public typealias SCNFloat = Float
#endif
```

Given this definition, this was intended as a plain conversion, not a bridge.